python/pyspark/sql/tests/test_functions.py

```diff
@@ -578,9 +588,13 @@ def test_approxQuantile(self):
         self.assertRaises(TypeError, lambda: df.stat.approxQuantile(["a", 123], [0.1, 0.9], 0.1))
 
     def test_sorting_functions_with_column(self):
-        from pyspark.sql import functions
         from pyspark.sql.column import Column
 
+        self.check_sorting_functions_with_column(Column)
+
+    def check_sorting_functions_with_column(self, tpe):
+        from pyspark.sql import functions
```

Comment:
no biggie but I believe we can just import this on the top

Reply:
Yeah, but it's imported in each test for some reason. Shall we do it in a separate PR?

Reply:
🆗

Reply:
oh, actually, it's already imported at the top of this file.